packages/ceramic-common/src/utils/doctype-utils.ts

```diff
@@ -141,7 +146,7 @@ export class DoctypeUtils {
 
         if (DoctypeUtils.isSignedRecordDTO(cloned)) {
             cloned.jws.link = new CID(cloned.jws.link)
-            cloned.linkedBlock = u8a.fromString(cloned.linkedBlock, 'base64')
+            cloned.linkedBlock = Buffer.from(cloned.linkedBlock, 'base64')
```

Comment:
Any reason to revert to using `Buffer` here please? Seems like using a `Uint8Array` would be a better option considering the changes we're making across all the packages, no?

Reply:
The API `ipfs.block.get` returns a `Block` instance that wraps a `Buffer`.  Until now we needed to deserialize a signed record DTO not serialize it, as well. That's why I used buffers in serialize/deserialize.